extension/ts/adapters/book/index.ts

```diff
@@ -0,0 +1,25 @@
+import {BookRecord} from "./types";
+import {getBooks} from "./bookFromSearch";
+import {getBooksFromWork} from "./bookFromWork";
+import {getBook} from "./bookFromEdition";
+import {setCache} from "./bookCache";
+
+const saveBook = async (book: BookRecord): Promise<void> => {
+	const body = new URLSearchParams({
+		form_id: book.id,
+		form_tags: book.tags.join(", "),
+	});
+	const response = await fetch("/ajax_changetags2.php", {method: "POST", body});
```

Comment:
is this the endpoint to submit the form?

Reply:
If you go to a book page, or even in the `Your books` overview, you can click the tags to edit them outside of the book edit page! There's an endpoint to submit _just_ the tags.